crates/ln-dlc-node/src/tests/dlc_channel.rs

```diff
@@ -244,19 +245,21 @@ async fn can_open_and_force_close_channel() {
         .unwrap();
 
     wait_until(Duration::from_secs(10), || async {
-        mine(1).await.unwrap();
-
         let dlc_channels = coordinator
             .dlc_manager
             .get_store()
             .get_signed_channels(None)?;
-        Ok(dlc_channels.is_empty().then_some(()))
+
+        Ok(dlc_channels
+            .iter()
+            .find(|dlc_channel| {
+                dlc_channel.counter_party == to_secp_pk_29(app.info.pubkey)
+                    && matches!(dlc_channel.state, SignedChannelState::SettledClosing { .. })
+            })
+            .cloned())
     })
     .await
     .unwrap();
-
```

Comment:
😂 you don't like that comment?

Reply:
I love it, but it doesn't actually make sense! This was actually closing a _settled_ channel, which does not involve CETs. So we don't need a Tardis after all :D